README.rst

```diff
@@ -31,6 +31,15 @@ To use setuptools_scm just modify your project's setup.py file like this:
            ...,
        )
 
+3. Access the version number in your package via :code:`pkg_resources`
+
+   E.g. (`PEP-0396 <https://www.python.org/dev/peps/pep-0396>`_):
+
+   .. code:: python
+
+      import pkg_resources
+      __version__ = pkg_resources.get_distribution(__name__).version
```

Comment:
I'm not sure what exception would be best to catch here, `pkg_resources.DistributionNotFound`?